.env.example

```
@@ -1,13 +1,21 @@
 REMOTE_DATABASE_URL=postgresql://postgres:postgres@localhost:5432/eurora
-JWT_SECRET=secret
+JWT_ACCESS_SECRET=secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_ACCESS_SECRET key should go before the REMOTE_DATABASE_URL key

(UnorderedKey)